include/sdf/Param.hh

```diff
@@ -401,6 +577,18 @@ namespace sdf
                                     const std::string &_valueStr,
                                     ParamVariant &_valueToSet) const;
 
+    /// \brief Method used to set the Param from a passed-in string
+    /// \param[in] _typeName The data type of the value to set
+    /// \param[in] _valueStr The value as a string
+    /// \param[out] _valueToSet The value to set
+    /// \param[out] _errors Vector of errors.
+    /// \return True if the value was successfully set, false otherwise
+    public: bool SDFORMAT_VISIBLE ValueFromStringImpl(
```

Comment:
This is an internal function (even though the symbol is visible), so we can just change the original instead of creating an overload.

Reply:
Oh ok, I was worried since it's `public` I could break something. I will remove the version without the `sdf::Errors` argument then.